cmake/vif.cmake

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2022 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Generates USB-C VIF policies in XML format from device tree.
+set(GEN_VIF_SCRIPT ${ZEPHYR_BASE}/scripts/generate_usb_vif/generate_vif.py)
+set(DTS_COMPATIBLE usb-c-connector)
+set(VIF_XML ${PROJECT_BINARY_DIR}/vif.xml)
+set(CMD_GEN_VIF ${PYTHON_EXECUTABLE} ${GEN_VIF_SCRIPT}
```

Comment:
please use lower case names for variables only intended for internal usage in this CMake file.
From what I can tell, that would be: `gen_vif_script`, `dts_compatible`, `vif_xml`, and `cmd_gen_vif`.

Reply:
Updated this in new revision